src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,436 @@
+
+Before I can use the Multi-Screen Window Placement API, I must ask the user for permission to do so.
+The new `window-placement` permission can be queried with the
+[Permissions API](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API) like so:
+
+```js
+let granted = false;
+try {
+  const { state } = await navigator.permissions.query({ name: "window-placement" });
+  granted = state === "granted";
+} catch {
+  // Nothing.
+}
+```
+
+The browser
+[can](https://webscreens.github.io/window-placement/#issue-c601b517:~:text=Permission%20check%20for%20above%3F,-partial)
+choose to show the permission prompt dynamically upon the first attempt to use any
+of the methods of the new API.
+Read on to learn more.
+
+### The `isMultiScreen()` method
+
+To use the the Multi-Screen Window Placement API, I will first call the
+`Window.isMultiScreen()` method. It returns a promise that resolves with either `true` or `false`,
+depending on whether one or multiple screens are currently connected to the machine. For my setup,
+it returns `true`.
+
+```js
+await window.isMultiScreen();
+// Returns `true` or `false`.
+```
+
+### The `getScreens()` method
+
+Now that I know that the current setup is multi-screen, I can obtain more
+information about the second screen using `Window.getScreens()`. It returns a promise
+that resolves with an array of `Screen` objects. On my MacBook Air 13 with a connected iPad, this
+returns an array of two `Screen` objects:
+
+```js
+await window.getScreens();
+/* Output from my MacBook Air 13″ with the iPad attached:
+  Screen 1 (built-in display):
+  availHeight: 975
+  availLeft: 0
+  availTop: 23
+  availWidth: 1680
+  colorDepth: 30
+  height: 1050
+  id: "0"
+  internal: true
+  left: 0
+  orientation: null
+  pixelDepth: 30
+  primary: true
+  scaleFactor: 2
+  top: 0
+  touchSupport: false
+  width: 1680
+
+  Screen 2 (iPad):
+  availHeight: 1001
+  availLeft: 1680
+  availTop: 23
+  availWidth: 1366
+  colorDepth: 24
+  height: 1024
+  id: "1"
+  internal: false
+  left: 1680
+  orientation: null
+  pixelDepth: 24
+  primary: false
+  scaleFactor: 2
+  top: 0
+  touchSupport: false
+  width: 1366
+*/
+```
+
+Note how the value of `left` for the iPad starts at `1680`, which is exactly the `width` of the
+built-in display. This allows me to determine exactly how the screens are arranged logically (next
+to each other, on top of each other, etc.). There is also data now for each screen to show whether
+it is an `internal` one and whether it is a `primary` one. Both also have an `id`, which, if
+persisted across browser sessions, allows for window arrangements to be restored.
+
+### The `onscreenschange` event
+
+The only thing missing now is a way to detect when my screen setup changes. A new event,
+`onscreenschange` (note: plural "screens"), does exactly that: it fires whenever the screen constellation
+is modified. This can happen when the resolution of one of the connected screens changes or when a
```

Comment:
Calling out that it is plural is a good idea, but I think it needs to be a bit more obvious. Because the new parenthetical interrupts the flow of the paragraph, I changed the beginning of the next sentence to call back more strongly to the previous thought.

Reply:
Applied. Much better now!